makingspace/ansible-rabbitmq
# fix: RabbitMQ repositories

## What does this PR do?
It updates the remote repositories used to install Rabbit MQ binaries since the previous ones (bintray) aren't available anymore.

The new repositories are hosted by: https://packagecloud.io/rabbitmq/rabbitmq-server/install

## How can I test it?

You can use this branch to test the vagrant provision it self:
1. Go to the vagrant-ubuntu repository
2. Edit the `requirements.yml` file by adding this branch to rabbitmq source:
    1. Add `version: smoreno/rabbitmq_fix` under "name: forked_rabbitmq" around line 15.
3. Make sure the vagrant machine does not exist.
4. Run the `startup.sh` script.
5. Validate the virtual machine is provisioned properly and rabbit get installed.

## Any relevant Ticket?
:ticket: [RabbitMQ is failing to get installed on local [ch62732]](https://app.clubhouse.io/makespace/story/62732/rabbitmq-is-failing-to-get-installed-on-local-envs)

## Any extra or background information?
